### ISSUE TITLE
`MathML-chemistry` `schema:accessibilityFeature` property not allowed

### ISSUE THREAD

**Issue by naglis-lab** [daisy/ace#459]
Using test EPUB (added .zip extension to upload to GitHub) [MathML-chemistry_test.epub.zip](https://github.com/user-attachments/files/27012103/MathML-chemistry_test.epub.zip) with Ace by DAISY version 1.4.5 we get a _Minor_ issue from `metadata-accessibilityfeature-invalid` rule:

- 'schema:accessibilityFeature' metadata must be set to one of the expected values 
-  Value 'MathML-chemistry' is invalid for 'schema:accessibilityFeature' metadata 
-  Use one of the metadata values defined by schema.org

<img width="1906" height="563" alt="Screenshot of Ace by DAISY version 1.4.5 report window with the results of MathML-chemistry_test.epub check" src="https://github.com/user-attachments/assets/a8ec3ebf-e3c7-4529-a4a2-6c8e9279040b" />

The `MathML-chemistry`value is in the vocabulary: https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20260326/#mathml-chemistry

Perhaps the allowed values should be updated?: https://github.com/daisy/ace/blob/e524df70bacd3df14af8a2a609ceb889ed5764b6/packages/ace-core/src/core/a11y-metadata.js#L150

Did not check if there are other missing accessibility feature values.

**Comment by clapierre** [comment]
This was added to our vocabulary in May of 2024, with [Issue 106](https://github.com/w3c/a11y-discov-vocab/issues/106), and [PR 107](https://github.com/w3c/a11y-discov-vocab/pull/107)
along with latex-chemistry.

**Comment by danielweck** [comment]
Thank you, I will update the vocabulary shortly.

**Comment by danielweck** [comment]
FYI:

Ace App (GUI) v1.4.6: https://github.com/daisy/ace-gui/releases/tag/v1.4.6
Ace (CLI) v1.4.6: https://github.com/daisy/ace/releases/tag/v1.4.6
